### PR TITLE
Added support for Feature Set context

### PR DESCRIPTION
**Pull request by clicktravel-basharat**

### CODE REVIEW

File: cheddar/cheddar-server/src/test/java/com/clicktravel/cheddar/server/http/filter/features/ContainerFeatureSetRequestFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.clicktravel.cheddar.request.context.features.FeatureSetContext;
+import com.clicktravel.cheddar.request.context.features.FeatureSetContextHolder;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ FeatureSetContextHolder.class })
+public class ContainerFeatureSetRequestFilterTest {
+
+    private static final String TRAVEL_CLOUD_FEATURE_SET_ID_HEADER = "Travel-Cloud-Feature-Set-Id";
```
Comment (clicktravel-steffan) [inline cheddar/cheddar-server/src/test/java/com/clicktravel/cheddar/server/http/filter/features/ContainerFeatureSetRequestFilterTest.java]: This identifier contains a marketing term (which is unstable) and is also specific to development at Click Travel and not applicable to a general audience. This identifier should instead be abstract, conveying its role. Perhaps `Active-Feature-Set` or `Enabled-Feature-Set`

**Comment by clicktravel-steffan** [comment]
Is feature set ID single valued or multi-valued (a set)?

**Comment by clicktravel-basharat** [comment]
The set is referred to by a single id which represents a set of features which are enabled.

**Comment by clicktravel-steffan** [comment]
@clicktravel-basharat Review complete, seems fine now

**Pull request merged**